src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/config/WebClientConfiguration.kt

```diff
@@ -57,9 +62,14 @@ class WebClientConfiguration(
     oAuth2AuthorizedClientService: ReactiveOAuth2AuthorizedClientService,
   ): ReactiveOAuth2AuthorizedClientManager {
     val authorizedClientProvider = ReactiveOAuth2AuthorizedClientProviderBuilder.builder().clientCredentials().build()
-    val authorizedClientManager =
-      AuthorizedClientServiceReactiveOAuth2AuthorizedClientManager(clientRegistrationRepository, oAuth2AuthorizedClientService)
-    authorizedClientManager.setAuthorizedClientProvider(authorizedClientProvider)
-    return authorizedClientManager
+    return AuthorizedClientServiceReactiveOAuth2AuthorizedClientManager(
+      clientRegistrationRepository,
+      oAuth2AuthorizedClientService,
+    ).also { it.setAuthorizedClientProvider(authorizedClientProvider) }
```

Comment:
It technically makes no difference - but - if you're only working on the context object and there is no return value then I think [apply](https://kotlinlang.org/docs/scope-functions.html#apply) is intended for suitable for this purpose. (I always think of also indicating a side effect to "something else").

Reply:
yes good call, will amend in next PR